robot_skills/src/robot_skills/arms.py

```diff
@@ -569,3 +569,199 @@ def _publish_marker(self, goal, color, ns = ""):
 
     def __repr__(self):
         return "Arm(side='{side}')".format(side=self.side)
+
```

Comment:
@Jip238 did you work on this? Please fix it ASAP, it's blocking the merge!

(in principle a new feature should *always* be committed in a feature branch so that it can be reviewed separately)

Reply:
I don't recall working on this. Could you take a look at it?

Reply:
Git blame: @Rayman 
I have cleaned up all the stuff which is duplicated.

Reply:
This is to get sergio working. All challenges except `robot.leftArm` to exist.